packages/devextreme-scss/scss/widgets/generic/tabPanel/layout/tab/_position-top.scss

```diff
@@ -13,8 +13,8 @@
 // adduse
 
 .dx-tabpanel-tabs-position-top {
-  .dx-tabpanel-tab {
```

Comment:
Hm, so it was earlier broken even for not nested tabPanel?)

Reply:
For non-nested cases, it was almost okay